test/libsolidity/semanticTests/structs/packed_storage_structs_delete.sol

```diff
@@ -0,0 +1,30 @@
+contract C {
+    struct str { uint8 a; uint16 b; uint8 c; }
+    uint8 x;
+    uint16 y;
+    str data;
+    function test() public returns (uint) {
+        x = 1;
+        y = 2;
+        data.a = 2;
+        data.b = 0xabcd;
+        data.c = 0xfa;
+        if (x != 1 || y != 2 || data.a != 2 || data.b != 0xabcd || data.c != 0xfa)
+            return 2;
+        delete y;
+        delete data.b;
+        if (x != 1 || y != 0 || data.a != 2 || data.b != 0 || data.c != 0xfa)
+            return 3;
+        delete x;
+        delete data;
+        return 1;
+    }
+    function checkIfDataIsEmpty() public returns (bool) {
+        return (x == 0 && y == 0 && data.a == 0 && data.b == 0 && data.c == 0);
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// test() -> 1
+// checkIfDataIsEmpty() -> true
```

Comment:
Sorry, I should have told you there is a special mechanism for this:
```suggestion
// storageEmpty -> 1
```
Can you undo the changes to the contract itself?

Reply:
Made these changes in the latest push.